vnext/Microsoft.ReactNative/Views/ReactRootControl.cpp

```diff
@@ -28,6 +28,21 @@
 #include <object/unknownObject.h>
 
 #include <ReactHost/MsoUtils.h>
+//#include <appmodel.h>
+
+extern "C" {
+
+typedef enum AppPolicyWindowingModel {
+  AppPolicyWindowingModel_None = 0,
+  AppPolicyWindowingModel_Universal = 1,
+  AppPolicyWindowingModel_ClassicDesktop = 2,
+  AppPolicyWindowingModel_ClassicPhone = 3
+} AppPolicyWindowingModel;
+
+WINBASEAPI
+_Check_return_ _Success_(return == ERROR_SUCCESS) LONG WINAPI
```

Comment:
why not include appmodel?

Reply:
The method is only exposed in appmodel.h for WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP) -- I'm open to suggestions. -- Xaml seems to call it all the time (in uwp and islands) -- I'm guessing XAML internally uses WINAPI_PARTITION_DESKTOP?